spec/rails_helper.rb

```diff
@@ -78,3 +78,7 @@
   #
   config.include ActiveSupport::Testing::TimeHelpers
 end
+
+# A dummy text to ensure testing without 'terms and conditions'.
+# Tests that depends on 'terms and conditions' will set them on example run.
+Rails.configuration.terms = false
```

Comment:
This ensures that tests will run without being influenced by the existence of `config/terms.md`.
Tests that are affected by the existence of terms will mock it's existence when needed.